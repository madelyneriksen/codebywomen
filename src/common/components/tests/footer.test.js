import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../footer';


describe("Footer Component", () => {
  it("Matches Snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})

