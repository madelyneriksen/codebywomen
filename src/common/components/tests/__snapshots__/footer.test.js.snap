// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Matches Snapshot 1`] = `
<footer
  className="h5 bg-light-gray flex flex-wrap justify-center"
>
  <span
    className="db w-100 tc self-center sans-serif"
  >
    Built with React and Gatsby.
  </span>
  <span
    className="db w-100 tc self-center sans-serif"
  >
    See the source code on 
    <a
      className="dark-gray"
      href="https://github.com/madelyneriksen/codebywomen"
    >
      Github.
    </a>
  </span>
  <span
    className="db w-100 tc self-end sans-serif bg-moon-gray pa1"
  >
    Copyright 2019 Madelyn Eriksen
  </span>
</footer>
`;
